pom.xml

```diff
@@ -30,19 +32,13 @@
         </license>
     </licenses>
 
-    <groupId>com.connexta.ingest</groupId>
+    <groupId>com.connexta.ion</groupId>
```

Comment:
✏️ Eric suggested `com.connexta.ion.ingest` for the `groupId`

Reply:
I'll change it again.